ATP.BusinessLogicLayer.Test/ServicesTest/BookingServiceTest.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Moq;
+using Microsoft.Extensions.Logging;
+using ATP.BusinessLogicLayer.Models;
+using AutoFixture;
+
+namespace ATP.BusinessLogicLayer.Test.ServicesTest;
+
+public class BookingServiceTest 
+{
+    private BookingService _service;
+    private Mock<ILogger<BookingService>> _loggerMock;
+    private string _csvFilePath;
+    private Fixture _fixture;
+    public BookingServiceTest()
+    {
+        _fixture = new Fixture();
+        _loggerMock = new Mock<ILogger<BookingService>>();
+        _csvFilePath = "test.csv";
+        _service = new BookingService(_csvFilePath, _loggerMock.Object);
+    }
+
+    [Fact]
+    public void BookFlight_AddsBooking_Successfully()
+    {
+        var flight = _fixture.Create<FlightDomainModel>();
+
+        _service.BookFlight(flight);
+
+        Assert.Single(_service.GetBookings());
+    }
+
+    [Fact]
+    public void BookFlight_Logs_Information()
+    {
+     
+        var flight = _fixture.Create<FlightDomainModel>();
+        string expectedMessage = $"Booking with ID {flight.Id} successfully created for the flight from {flight.DepartureCountry} to {flight.DestinationCountry} on {flight.DepartureDate}.";
+
+        _service.BookFlight(flight);
+
+        _loggerMock.Verify(
+            logger => logger.Log(
+                LogLevel.Information,
+                It.IsAny<EventId>(),
+                It.Is<It.IsAnyType>((v, t) => string.Equals(expectedMessage, v.ToString(), StringComparison.OrdinalIgnoreCase)),
+                It.IsAny<Exception>(),
+                (Func<It.IsAnyType, Exception, string>)It.IsAny<object>()),
+            Times.Once);
+    }
+
+    [Fact]
+    public void CancelBooking_RemovesBooking_Successfully()
+    {
+        var flight = _fixture.Create<FlightDomainModel>();
+        _service.BookFlight(flight);
+        int bookingId = flight.Id;
+
+        _service.CancelBooking(bookingId);
+
+        Assert.Empty(_service.GetBookings());
+    }
+
+    [Fact]
+    public void GetBookings_Returns_Bookings_List()
+    {
+        var flight1 = _fixture.Create<FlightDomainModel>();
+        var flight2 = _fixture.Create<FlightDomainModel>();
+        _service.BookFlight(flight1);
+        _service.BookFlight(flight2);
+
+        var bookings = _service.GetBookings();
+
+        Assert.Collection(bookings,
+            booking => Assert.Equal(flight1.Id, booking.FlightId),
+            booking => Assert.Equal(flight2.Id, booking.FlightId));
```

Comment:
Split assertion here may be more readable
```suggestion
        Assert.Equal(2, bookings.Count);
    Assert.Contains(bookings, booking => booking.FlightId == flight1.Id);
    Assert.Contains(bookings, booking => booking.FlightId == flight2.Id);
```